app.py

```diff
@@ -23,10 +23,25 @@ def init_app(self):
         """
         Called as the application is being initialized
         """
-
         tk_multi_breakdown = self.import_module("tk_multi_breakdown")
         cb = lambda : tk_multi_breakdown.show_dialog(self)
         self.engine.register_command("Scene Breakdown...", cb, { "short_name": "breakdown" })
 
 
-
+    def show_ui(self):
```

Comment:
I'd prefer something more specific just in case we ever add multiple ui's to this app?